### ISSUE TITLE
Chore: Add nightly builds

### ISSUE THREAD

**Issue by gruuya** [GlareDB/glaredb#1930]
## Description

This will spare people who want to work with the latest GlareDB version from having to build from source, and thus smooth out the adoption.

**Comment by scsmithr** [comment]
We did have this previously, but I toned it back to only running on Mondays instead of every day in https://github.com/GlareDB/glaredb/pull/1881. If we want to, we can bump that back up to every day.

**Comment by gruuya** [comment]
Oh I see. Yeah weekly is also fine, it's just that I don't see an immediate/low-friction way on how to get those binaries. One needs to go to GitHub Actions and pick the latest `Build Release` run to reach them. And even then you won't be able to download those artifacts unless you're logged in to GitHub.

So perhaps it's worth just configuring https://nightly.link/ to point to those binaries (example [here](https://github.com/splitgraph/seafowl#nightly-builds))?

**Comment by djouallah** [comment]
yes please do, it does not make much sense to wait months to test a new functionality, duckdb and databend nightly build is very convenient !! even weekly is fine, and please no github shenanigans
